renderer/reducers/activity.js

```diff
@@ -279,7 +294,11 @@ const applySearch = (data, searchText) => {
         item =>
           (item.tx_hash && item.tx_hash.includes(searchText)) ||
           (item.payment_hash && item.payment_hash.includes(searchText)) ||
-          (item.payment_request && item.payment_request.includes(searchText))
+          (item.payment_preimage && item.payment_preimage.includes(searchText)) ||
```

Comment:
consider extracting this into a helper to reduce potential typo errors. Smth like
`const matches = (prop) => item[props] &&  item[prop].includes(searchText)`
and then 
`['text_has','payment_hash','payment_preimage',...].some(matches)`

Reply:
Yes, good idea. Have refactored this.